Dockerfile

```diff
@@ -1,6 +1,27 @@
-FROM alpine:3.7
-MAINTAINER Nic Cope <n+docker@rk0n.org>
+FROM node as node
+ADD frontend/ .
+RUN npm install && npm run build
+
+FROM golang as golang
+WORKDIR /go/src/github.com/negz/kuberos/
+ENV GOOS="linux" GOARCH="amd64" CGO_ENABLED=0
```

Comment:
I don't think the `GOOS` and `GOARCH` should be necessary now that we're building in Docker.

Reply:
That's right. I tested that only GCO_ENABLED=0 is necessary. Fixing that in upcoming commit.